.github/workflows/release_builds.yml

```diff
@@ -279,7 +292,7 @@ jobs:
       - name: Store binary as action artifact
         uses: actions/upload-artifact@v4
         with:
-          name: ${{matrix.arch_os}}_fips
+          name: "${{matrix.arch_os}}-fips"
```

Comment:
Not sure why this is changed, but I think if you change this then you also need to change the artifact name in `Build container` job's `Download binary action artifact from build phase (FIPS)` step below.

https://github.com/SumoLogic/sumologic-otel-collector/pull/1416/files#diff-e048a1bb70d286fe06c2fdaa5a9ad1890a305dcea7eb3894e82599cd4c7e0101R352